hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -94,6 +95,48 @@ public EthTxData replaceCallData(byte[] callData) {
                 s);
     }
 
+    @VisibleForTesting
+    EthTxData replaceOfferedGasPrice(@NonNull final BigInteger replacementGasPrice) {
```

Comment:
These methods look useful but are these just added for completeness as they do not seem to be used currently except in tests?

Reply:
Just for ease of testing.